[DRAFT] Responsive update (#4620)
aws-amplify/docs: src/pages/console/uibuilder/responsive.mdx

}
```

> Breakpoints are set at the Theme level, which means they apply to all components in your Figma file.  You can customize these breakpoints in the Amplify Theme Editor plugin:

Reviewer: ```suggestion
Breakpoints are set at the [Amplify UI Theme](https://ui.docs.amplify.aws/react/theming/responsive) level, which means they apply to all components in your Figma file. You can customize these breakpoints in the Amplify Theme Editor plugin:
```

> 1. Right click in your Figma project and select "Plugins" > "AWS Amplify Theme Editor" (screenshot)

Reviewer: ```suggestion
1. Right-click in your Figma project and select "Plugins" > "AWS Amplify Theme Editor" (screenshot)
```

> 2. Select the "Breakpoints" Tab

Reviewer: ```suggestion
2. Select the "Breakpoints" tab
```

(screenshot)
3. Click on any breakpoint to customize the pixel value
(screenshot)

## 2. Configure your Figma variants

> To make your components responsive, you first need a component with variants.  [Learn more about creating variants in Figma's documentation](https://help.figma.com/hc/en-us/articles/360061175334-Create-interactive-components-with-variants).

Reviewer: I'm not sure if this is the best way to lead into this section. Maybe better:
```suggestion
To make a component responsive, first create different variants for the component based on the breakpoints. [Learn more about creating variants in Figma's documentation](https://help.figma.com/hc/en-us/articles/360061175334-Create-interactive-components-with-variants).
```
